garak/buffs/paraphrase.py

```diff
@@ -69,9 +70,12 @@ def transform(
             yield paraphrased_attempt
 
 
-class Fast(Buff):
+class Fast(Buff, HFCompatible):
     """CPU-friendly paraphrase buff based on Humarin's T5 paraphraser"""
 
+    DEFAULT_PARAMS = Buff.DEFAULT_PARAMS | {
+        "hf_args": {"device": "cpu", "torch_dtype": "float32"}
+    }
```

Comment:
Should this be refactored to match pattern used in `atkgen`?
```suggestion
    DEFAULT_PARAMS = Buff.DEFAULT_PARAMS | {
        "para_model_name": "garak-llm/chatgpt_paraphraser_on_T5_base",
        "para_model_config": {
            "hf_args": {"device": "cpu", "torch_dtype": "float32"},
        }
    }
```

The model instantiates using `AutoModelForSeq2SeqLM` so I am not sure if a `para_model_type` make sense as `generators.huggingface.*` does not provide a class I think can be substituted for this currently and may require a more abstract rethink of how this buff is applied.

Reply:
The comment was a but of reflection on previous expectations and discussion, by exposing the `name` and `type` we gain flexibility and some consistency with `cli` where objects that instantiate a model require a `name + type + config` combination in a similar format.

As we work on documenting the config patterns better maybe this will become more clear on where consistency is useful vs not.

Reply:
the model should be exposed as a config param somewhere and I somewhat prefer that to be a value rather than a key

Reply:
```suggestion
    DEFAULT_PARAMS = Buff.DEFAULT_PARAMS | {
        "para_model_name": "garak-llm/chatgpt_paraphraser_on_T5_base",
        "hf_args": {"device": "cpu", "torch_dtype": "float32"}
    }
```

How about we split the difference and expose the model name but retain hf config map at top level inferring this class only supports huggingface models at this time, if we expand support later we can consider it a breaking change for config of this specific class.

Reply:
This is consistent with `HFDetectors`. Going this route for this iteration and we can revisit later.